DigitalOffice/Tests/CompanyServiceTests/Mappers/PositionMapperTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using LT.DigitalOffice.CompanyService.Mappers;
+using LT.DigitalOffice.CompanyServiceUnitTests.UnitTestLibrary;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.CompanyServiceUnitTests.Mappers
+{
+    internal class PositionMapperTests
+    {
+        private PositionMapper mapper;
+
+        [SetUp]
+        public void SetUp()
```

Comment:
OneTimeSetUp

Reply:
Why?

Reply:
Why do you need to call and initialize variables in SetUp repeatedly?

Reply:
Ok, it's really not needed.

Reply:
fixed